plugins/@grouparoo/hubspot/src/lib/export-contacts/exportRecord.ts

```diff
@@ -180,6 +179,7 @@ async function syncAssociations(
   client: HubspotClient,
   contact: any,
   companyKey: string,
+  companyRecordField: string,
```

Comment:
but it's not actually using it?

Reply:
actually, the way I did this time it's not using `Company.Name`, it's using like: `oldRecordProperties[companyRecordField]`.